config/google_cloud_exporter/windows_events/config.yaml

```diff
@@ -0,0 +1,28 @@
+receivers:
+  windowseventlog/application:
+    channel: application
```

Comment:
Does casing matter here? I'm looking at the xml for an event now and channel says

```xml
<Channel>Application</Channel>
```